src/features/lightspeed/base.ts

```diff
@@ -132,7 +132,7 @@ export class LightSpeedManager {
         )) as {
           copilot: { enable?: { ansible?: boolean } };
         };
-        const copilotEnableForAnsible = githubConfig?.copilot?.enable?.ansible;
```

Comment:
@tanwigeetika1618 I think the change on this line is causing the UI test failure. Would you try to re-run the test after reverting this change?

In my local test runs, I could recreate the isue with this change and the test started succeeding after I reverted the change. Though I have not tried tests using CI, hopefully this would resolve the issue.